content/en/blog/_posts/2023-10-20-kcs-shanghai/index.md

```diff
@@ -0,0 +1,113 @@
+---
+layout: blog
+title: "A Quick Recap of 2023 China Kubernetes Contributor Summit"
+slug: kcs-shanghai
+date: 2023-10-20
+---
+
```

Comment:
As this is a [mirrored](https://github.com/kubernetes/contributor-site/pull/444) blog article, we should consider including `canonicalUrl` in the front matter.
```suggestion
canonicalUrl: https://www.kubernetes.dev/blog/2023/10/20/kcs-shanghai/
---

```